resources/zib/zib-wounds.WoundCharacteristics.xml

```diff
@@ -0,0 +1,286 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-wounds.WoundCharacteristics" />

```

Comment:
The DateOfLastDressingChange and Extent concepts are still missing here.

Reply:
Added